mmv1/third_party/terraform/utils/compute_instance_helpers.go.erb

```diff
@@ -346,6 +346,29 @@ func flattenConfidentialInstanceConfig(ConfidentialInstanceConfig *computeBeta.C
 	}}
 }
 
+func expandAdvancedMachineFeatures(d TerraformResourceData) *computeBeta.AdvancedMachineFeatures {
+	if _, ok := d.GetOk("advanced_machine_features"); !ok {
+		return nil
+	}
+
+	prefix := "advanced_machine_features.0"
+	return &computeBeta.AdvancedMachineFeatures{
+		EnableNestedVirtualization: d.Get(prefix + ".enable_nested_virtualization").(bool),
+		ThreadsPerCore:             int64(d.Get(prefix + ".threads_per_core").(int)),
+		//	ForceSendFields:           []string{"EnableSecureBoot"},
```

Comment:
Think this was copied over from another block.

Reply:
Removed it. CVM had it too so I removed that as well